specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -759,22 +944,285 @@
+            "schema": {
+              "$ref": "#/definitions/ReplicationDetailsList"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/providers/Microsoft.ADHybridHealthService/addsservices/{serviceName}/replicationstatus": {
+      "get": {
+        "tags": [ "Adds" ],
+        "description": "Gets Replication status for a given Active Directory Domain Service, that is onboarded to Azure Active Directory Connect Health.",
+        "x-ms-examples": {
+          "addsServices_getReplicationStatus": {
+            "$ref": "./examples/ReplicationStatus.json"
+          }
+        },
+        "operationId": "addsServices_getReplicationStatus",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": " ",
+            "schema": {
+              "$ref": "#/definitions/ReplicationStatus"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.ADHybridHealthService/addsservices/{serviceName}/replicationsummary": {
+      "get": {
+        "tags": [ "Adds" ],
+        "description": "Gets complete domain controller list along with replication details for a given Active Directory Domain Service, that is onboarded to Azure Active Directory Connect Health.",
+        "x-ms-examples": {
+          "addsServices_listReplicationSummary": {
+            "$ref": "./examples/ReplicationSummary.json"
+          }
+        },
+        "operationId": "addsServices_listReplicationSummary",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "description": "The server property filter to apply.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "isGroupbySite",
+            "in": "query",
+            "description": "Indicates if the result should be grouped by site or not.",
+            "required": true,
+            "type": "boolean"
+          },
+          {
+            "name": "query",
+            "in": "query",
+            "description": "The custom query.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "nextPartitionKey",
+            "in": "query",
+            "description": "The next partition key to query for.",
+            "required": true,
+            "type": "string",
+            "enum": [ " " ],

```

Comment:
Please create a model definition and refer its schema for enum parameters

Reply:
Left this out, as mentioned above.

Reply:
The way the enum is modeled here is not the issue. We discourage usage of inline definitions of schemas (which is enum here). Please follow an example [here](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/analysisservices/resource-manager/Microsoft.AnalysisServices/stable/2017-08-01/analysisservices.json#L99) and move the schema definition into definitions section

Reply:
Got it now. Will send another iteration with the change.

Reply:
@dsgouda , I made the changes for the enum definition as suggested by you. But after that build is failing:
https://travis-ci.org/Azure/azure-rest-api-specs/jobs/389568900

Can you please help me understand what am I doing wrong here? How can I correct that? Thanks for helping.